x-pack/plugins/fleet/server/errors/index.ts

```diff
@@ -43,6 +43,7 @@ export class PackageInvalidArchiveError extends FleetError {}
 export class PackageRemovalError extends FleetError {}
 export class PackageESError extends FleetError {}
 export class ConcurrentInstallOperationError extends FleetError {}
+export class PackageSavedObjectConflictError extends FleetError {}
```

Comment:
Since this error replaces `ConcurrentInstallOperationError` that [returns a 409](https://github.com/elastic/kibana/blob/0a90b66ed37936086ce272424124fa51ee98aa14/x-pack/plugins/fleet/server/errors/handlers.ts#L100-L102) maybe we should be explicitly define the error code to return in this case as well? Otherwise by default it might go to a 500 (I think)

Reply:
We default to 400 here, but I've update to a 409 as you've suggested since it's much more appropriate.